projects/online/online/utils/gdb.py

```diff
@@ -70,45 +68,55 @@ def submit(self, event: Event):
     def submit_low_latency_pe(
         self,
         result: bilby.core.result.Result,
-        mollview_map: "np.ndarray",
-        skymap: "table.Table",
+        skymap: "BinTableHDU",
         graceid: int,
         event_time: float,
     ):
         event_dir = self.write_dir / f"event_{int(event_time)}"
+        event_dir.mkdir(exist_ok=True)
```

Comment:
Does this line need to be here? The directory should be getting created by the `event.write` call in the event subprocess.